src/styles/_catalog.scss

```
@@ -1,6 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(auto-fill, 300px);
+  gap: 48px;
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, 200px);
+  }
```

```suggestion
  grid-template-columns: repeat(1, 200px);
  grid-template-rows: repeat(auto-fill, 300px);
  gap: 48px;
```
Wouldn't it work the same?

---

sorry ) I don't quite understand, "grid-template-columns: repeat(1, 200px);" it applies for resolution and other  "grid-template-rows: repeat(auto-fill, 300px);
  gap: 48px;" applie for all @media queries, as I understand